sdk/src/main/java/io/coti/sdk/TokenManagement.java

```diff
@@ -0,0 +1,161 @@
+package io.coti.sdk;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import io.coti.basenode.data.*;
+import io.coti.basenode.exceptions.CotiRunTimeException;
+import io.coti.basenode.http.*;
+import io.coti.basenode.http.data.TokenGenerationFeeResponseData;
+import io.coti.sdk.utils.CryptoUtils;
+import io.coti.sdk.utils.Mapper;
+import lombok.experimental.UtilityClass;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.time.Instant;
+
+@UtilityClass
+public class TokenManagement {
```

Comment:
management? this is more of a util that calls token APIs. let think on a new name maybe TokenUtilities